substrate/frame/broker/src/utility_impls.rs

```diff
@@ -60,6 +60,10 @@ impl<T: Config> Pallet<T> {
 	}
 
 	pub fn sale_price(sale: &SaleInfoRecordOf<T>, now: RelayBlockNumberOf<T>) -> BalanceOf<T> {
+		if let Some(override_price) = OverridePrice::<T>::get() {
+			return override_price;
+		}
+	
```

Comment:
This removes the entire lead-in period and sets the price throughout the sale to be the minimum price, which is not what is intended.

> The UX would likely be better if the end_price was only updated as the sale rotates, instead of straight away. Therefore a storage item which is an option could override the amount passed to the price adapter.

overriding the amount to the price adapter when the sale is rotated, as in when `adapt_price` is called, not `leadin_factor_at`